scripts/management/xcash-maintenance

```diff
@@ -0,0 +1,264 @@
+#!/bin/bash
+
+# Adjust this variable to the name of your xcash-daemon unit file name
+XCASH_DAEMON_SRV=xcash-daemon
+
+# Adjust this variable to the name of your xcash-wallets unit file name
+XCASH_WALLET_SRV=xcash-rpc-wallet
+
+# Adjust this variable to the name of your xcash-dpops unit file name
+XCASH_DPOPS_SRV=xcash-dpops
+
+# Adjust this variable to the name of your mongodb unit file name
+MONGODB_SRV=mongodb
+
+XCASH_DAEMON_LOG_FILE=/var/x-network/logs/blockchain.log
+XCASH_WALLET_LOG_FILE=/var/x-network/logs/rpc.log
+### CUSTOM VARIABLES ###
+
+DAEMON_URL="http://127.0.0.1:18281/json_rpc"
+WALLET_URL="http://127.0.0.1:18285/json_rpc"
+DPOPS_QUERY="curl -s -H \"Content-Type: application/json\" -X GET http://127.0.0.1:18283"
+CURL='curl -w "\n" -s -H "Content-Type: application/json" -X'
+
+red=`tput setaf 1`
+green=`tput setaf 2`
+reset=`tput sgr0`
+
+PS3='
+Please enter an option: '
+
+options=(
+	"Check blockchain height"
+	"Get node statistics"
+	"Get node balance"
+	"Check services status"
+	"Start stopped services"
+    "Stop all services"
+	"Restart all services"
+	"Get X-Cash DPoPS log"
+    "Get X-Cash Daemon log"
+    "Get X-Cash RPC Wallet log"
+	"Quit"
+)
+
+function check_height(){
+	if lsof -Pi :18281 -sTCP:LISTEN -t >/dev/null ; then
+        DATA=`$CURL GET $DAEMON_URL -d '{"jsonrpc":"2.0","id":"0","method":"get_block_count"}'`
+
+		HEIGHT=$(echo "$DATA" | grep '"count"' | awk {'print $2'} | sed s'|,||g' | tr -d $'\r')
+		STATUS=$(echo "$DATA" | grep '"status"' | awk {'print $2'} | sed s'|"||g' | tr -d $'\r')
+		
+		if [ $STATUS == "OK" ]; then
+			STATUS=${green}OK${reset}
+		else 
+			SATTUS=${red}FAIL${reset}
+		fi
+
+		echo
+		echo "Blockhain sync status: $STATUS"
+		echo "Blockchain height: $HEIGHT"
+	else
+		echo
+		echo "X-Cash Daemon service is not running...."
+		echo "Check the $XCASH_DAEMON_SRV service status"
+	fi
+}
+
+function get_node_balance(){
+	if lsof -Pi :18285 -sTCP:LISTEN -t >/dev/null ; then
+		DATA=`$CURL GET $WALLET_URL -d '{"jsonrpc":"2.0","id":"0","method":"get_balance","params":{"account_index":0,"address_indices":[0]}}'`
+		
+		ADDR=$(echo "$DATA" | grep '"address"' | awk {'print $2'} | sed s'|"||g' | sed s'|,||g' | head -1 | tr -d $'\r')
+		UNSPENT_OUTPUTS=$(echo "$DATA" | grep '"num_unspent_outputs"' | awk {'print $2'} | sed s'|"||g' | sed s'|,||g' | head -1 | tr -d $'\r')
+
+		ATOMIC_TOTAL_BALANCE=$(echo "$DATA" | grep '"balance"' | awk {'print $2'} | sed s'|,||g' | head -1 | tr -d $'\r')
+		ATOMIC_UNLOCKED_BALANCE=$(echo "$DATA" | grep '"unlocked_balance"' | awk {'print $2'} | sed s'|,||g' | head -1 | sed s"|^M||g" | tr -d $'\r')
+		TOTAL_BALANCE=$((ATOMIC_TOTAL_BALANCE/1000000))
+		UNLOCKED_BALANCE=$((ATOMIC_UNLOCKED_BALANCE/1000000))
```

Comment:
Maybe do some kind of global constant for the 100000 since you use it twice? and use it in multiple functions `readonly MYVARNAME=1000000`?